cirq/google/engine/engine_processor.py

```diff
@@ -223,7 +224,7 @@ def remove_reservation(self, reservation_id: str):
             raise ValueError('Cannot determine freeze_schedule from processor.'
                              'Call _cancel_reservation or _delete_reservation.')
         secs_until = (reservation.start_time.seconds -
-                      int(datetime.datetime.now().timestamp()))
+                      int(datetime.datetime.now(tz=utc).timestamp().timestamp()))
```

Comment:
Why are there two timestamps() here?

Reply:
Apparently I hadn't pushed the fix for this. \Done.